requirements.txt

```diff
@@ -14,6 +14,7 @@ cu2qu==1.6.7              # via ufo2ft
 fonttools[ufo]==4.13.0    # via booleanoperations, cffsubr, compreffor, cu2qu, nanoemoji (setup.py), ufo2ft, ufolib2
 fs==2.4.11                # via fonttools
 lxml==4.5.1               # via nanoemoji (setup.py), picosvg
+ninja==1.10.0.post1
```

Comment:
ninja should be added to the setup.py `install_requires` list as well (so that it's pulled automatically when one does `pip install nanoemoji`). The requirements.txt is auto-generated via `pip-compile setup.py`

Reply:
`pip-compile` script is part of `pip-tools` https://github.com/jazzband/pip-tools

We should add `pip-tools` to the dev-requirements.in file and regenerate `dev-requirements.txt` from it (with `pip-compile dev-requirements.in`)

Reply:
Fixed.